### PR TITLE
fixes for opening map from address search (fix #7386) (fix #7387)

### LINKED ISSUE #7386: Open map from address search shows routing line
Testing the change from #7228 (Add icon to address search to show on map) I noticed that on the opened map the routing line with beginn or target(?) to map center will be shown on the opened live map.

I don't think this is useful and helpful and maybe not even intentionally done?!

### LINKED ISSUE #7387: Live map opened from address search cannot be disabled
##### Detailed steps causing the problem:
- Open search menu
- Search for a location/address
- Click map icon to open the map

##### Actual behavior after performing these steps:
The map menu does not contain the menu item "disable live" as it does on the "normal" live map

##### Expected behavior after performing these steps:
"Disable live" should be working the same on this map as on the "normal" live map

##### Version of c:geo used:
2019.02.23

##### Is the problem reproducible for you?
Yes

### PR DESCRIPTION
**Pull request by moving-bits**
don't show direction line when called from address search
show "enable live / disable live" menu entry when called from address search

### CODE REVIEW

File: main/src/cgeo/geocaching/maps/CGeoMap.java [hunk 1]
```diff
@@ -423,6 +423,12 @@ public void onCreate(final Bundle savedInstanceState) {
 
         ArrayList<Location> trailHistory = null;
 
+        final Geopoint savedCoords = mapOptions.coords;  // remember for centerMap
+        if ((mapOptions.mapMode == MapMode.LIVE) && (mapOptions.coords != null)) {
```
Comment (UniQP) [inline main/src/cgeo/geocaching/maps/CGeoMap.java]:
```suggestion
        if (mapOptions.mapMode == MapMode.LIVE && savedCoords != null) {
```

File: main/src/cgeo/geocaching/maps/CGeoMap.java [hunk 2]
```diff
@@ -423,6 +423,12 @@ public void onCreate(final Bundle savedInstanceState) {
 
         ArrayList<Location> trailHistory = null;
 
+        final Geopoint savedCoords = mapOptions.coords;  // remember for centerMap
```
Comment (UniQP) [inline main/src/cgeo/geocaching/maps/CGeoMap.java]: Is it possible to move the whole block down to fix the codacy issue?

Comment (moving-bits) [reply main/src/cgeo/geocaching/maps/CGeoMap.java]: done

File: main/src/cgeo/geocaching/maps/CGeoMap.java (hunk 1 above)
Comment (moving-bits) [reply main/src/cgeo/geocaching/maps/CGeoMap.java]: done

**Pull request merged**